curvefs/proto/topology.proto

```diff
@@ -328,6 +328,7 @@ message MetaServerLocation {
     required string hostIp = 2;
```

Comment:
here change it to internal ip, internal port

Reply:
> here change it to internal ip, internal port

Change to internalIp and internalPort on all the related interfces.